Taarafo.Portal.Web/Services/Foundations/Comments/CommentService.Validations.cs

```diff
@@ -31,6 +31,11 @@ private static void ValidateCommentIsNotNull(Comment comment)
             }
         }
 
+        public static void ValidateCommentId(Guid commentId)
+        {
+            Validate((Rule: IsInvalid(commentId), Parameter: nameof(Comment.Id)));
```

Comment:
Fat arrow this.
for any function that has only one line we replace the scoped brackets with a `=>` fat-arrow. thanks.

Reply:
@hassanhabib done